cli/determined_cli/experiment.py

```diff
@@ -273,14 +273,20 @@ def submit_experiment(args: Namespace) -> None:
 
 
 def local_experiment(args: Namespace) -> None:
-    experiment_config = _parse_config_file_or_exit(args.config_file)
-
     try:
         from determined import experimental, load
     except ImportError as e:
         print("--local requires that the `determined` package is installed.")
         raise e
 
+    if not args.test_mode:
+        raise NotImplementedError(
+            "--local mode without --test mode is not yet implemented. Please either add the "
```

Comment:
How about:
```
        raise NotImplementedError(
            "Local training mode (--local mode without --test mode) is not yet implemented. Please "
            "try local test mode by adding the --test flag or cluster training mode by removing "
            "the --local flag."
        )
```
?

Reply:
I mean "not yet implemented" sounds more like an error message for our engineers rather than users. It sounds like Determined give me a partially finished product.

Reply:
I think it is important to communicate to users that we intend to do it.  Right now we support 3 combinations but have an API for four... I'd be very confused as a user if the message made it sound like `--local` without `--test` was not supported *on purpose*.  I would totally get it if it was just not supported *yet*.

What about "not yet supported"?  Would that be better?

Reply:
yes. It sounds like we are setting an expectation here. Not sure if we actually going to do it soon. But "not yet supported" is fine.

Reply:
done!  I like `supported` better too, I think.